### PR TITLE
Clone specific version of Midas

### PR DESCRIPTION
**Pull request by jwvanderbeck**
stable-diffusion-webui's env is not compatible with 3.1 of Midas

### CODE REVIEW

**Comment by Extraltodeus** [comment]
thank you

**Comment by Snowy9199** [comment]
so I am extremely new to all this, how do you use the @jwvanderbeck
[Clone specific version of Midas](https://github.com/Extraltodeus/depthmap2mask/pull/44/commits/7c059cd58d651dfba4713fbb3e9c63d5f5555bd8) ? I cant seem to download it and it doesn't seem to be a command line prompt. do i need to copy it to a text file and run it using visual studio code? does it matter which folder i run it in? do I need to use a certain program? do i add it as a file somewhere? do i add the instructions to an existing flie?  I don't really know how to google my way out of this one because I'm unsure what to ask so ELI5 please.

**Comment by jwvanderbeck** [comment]
The project already merged in my fixes, so you don't need to do anything except be on the latest version.  If you are "stuck" on an older version before this was fixed and can't get updated, the easiest thing is probably to simply delete and reinstall
